modules/apps/portal-workflow/portal-workflow-test/src/testFunctional/tests/WorkflowMessageBoards.testcase

```diff
@@ -133,6 +133,123 @@ definition {
 			threadSubject = "MB Thread Subject");
 	}
 
+	@description = "This is a test for LRQA-69699. Verify that the statics are updated. "
+	@ignore = "Test Stub"
+	@priority = "3"
+	test AssertStaticsAreUpdated {
+
+		// TODO LRQA-69699 AssertStaticsAreUpdated pending implementation
+
+	}
+
+	@description = "This is a test for LRQA-69700. Verify that the user can approve a thread."
+	@ignore = "Test Stub"
+	@priority = "3"
+	test CanApproveAThread {
+
+		// TODO LRQA-69700 CanApproveAThread pending implementation
+
+	}
+
+	@description = "This is a test for LRQA-69693. Verify if the user can assign a Workflow. "
+	@ignore = "Test Stub"
+	@priority = "3"
+	test CanAssignAWorkflow {
+
+		// TODO LRQA-69693 CanAssignAWorkflow pending implementation
+
+	}
+
+	@description = "This is a test for LRQA-69702. Verify the user can attach files to a thread. "
+	@ignore = "Test Stub"
+	@priority = "3"
+	test CanAttachFiles {
+
+		// TODO LRQA-69702 CanAttachFiles pending implementation
+
+	}
+
+	@description = "This is a test for LRQA-69698. Verify that a Thread can be Locked. "
+	@ignore = "Test Stub"
+	@priority = "3"
+	test CanBeLocked {
+
+		// TODO LRQA-69698 CanBeLocked pending implementation
+
+	}
+
+	@description = "This is a test for LRQA-69692. Assert that user can clear search bar results. "
+	@ignore = "Test Stub"
+	@priority = "3"
+	test CanClearSearchBarResults {
+
+		// TODO LRQA-69692 CanClearSearchBarResults pending implementation
+
+	}
+
+	@description = "This is a test for LRQA-69697. Verify thath the user can filter the results by Categories. "
+	@ignore = "Test Stub"
+	@priority = "3"
+	test CanFilterByCategory {
+
+		// TODO LRQA-69697 CanFilterByCategory pending implementation
+
+	}
+
+	@description = "This is a test for LRQA-69696. Verify that the user can filter the results by thread. "
+	@ignore = "Test Stub"
+	@priority = "3"
+	test CanFilterByThread {
+
+		// TODO LRQA-69696 CanFilterByThread pending implementation
+
+	}
+
+	@description = "This is a test for LRQA-69691. Verify if the user can get search bar results. "
+	@ignore = "Test Stub"
+	@priority = "3"
+	test CanGetSearchBarResults {
+
+		// TODO LRQA-69691 CanGetSearchBarResults pending implementation
+
+	}
+
+	@description = "This is a test for LRQA-69701. Verify that the user can reject a thread. "
+	@ignore = "Test Stub"
+	@priority = "3"
+	test CanRejectAThread {
+
+		// TODO LRQA-69701 CanRejectAThread pending implementation
+
+	}
+
+	@description = "This is a test for LRQA-69703. Assert that user can reply to the thread. "
+	@ignore = "Test Stub"
+	@priority = "3"
+	test CanReplyToThread {
+
+		// TODO LRQA-69703 CanReplyToThread pending implementation
+
+	}
+
+	@description = "This is a test for LRQA-69705. Assert that the user can create a thread within a category. "
+	@ignore = "Test Stub"
+	@priority = "3"
+	test CreateAThreadWithinACategory {
```

Comment:
Hi Kristin! This scenario does not belong to Workflow scope. Could you, please, remove this stub? Thank you! :)

Reply:
Removed this test stub!